.github/workflows/ci.yml

```diff
@@ -58,6 +58,29 @@ jobs:
         ctest-options: -V
         configure-options: -DCPR_ENABLE_LINTING=ON
 
+  cppcheck:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.2.0
+      with:
+        submodules: true
+    - name: Update package list
+      run: sudo apt update
+    - name: Install cppcheck
+      run: sudo apt install cppcheck
```

Comment:
Github complains about `apt`:
```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```
Maybe we should exchange it with `apt-get`.

Reply:
I would keep it that way since we are not using any complex apt operations and in my eyes this is the more further proof command.